sycl/test-e2e/OnlineCompiler/online_compiler_OpenCL.cpp

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: opencl, opencl_icd, cm-compiler
-// XFAIL: gpu
+// XFAIL: gpu || cpu
```

Comment:
> @intel/dpcpp-sanitizers-review Ping, needed to disable some tests that started failing in the GPU driver, please take a look, thanks!

@sarnex  I'm a bit confused, this is the only change I see since my last review, how does xfailing in CPU relate to problems in the GPU driver?

Reply:
yeah youre right this makes no sense, let me remove it try to remember why i added it, thanks

Reply:
ok it took me forever but i investigated it, basically the test calls the gpu driver even in cpu mode, that's why it's failing, so im gonna add the XFAIL back

Reply:
Is that expected? Should we rewrite the test not to do that?